packages/ddp-server/livedata_server_tests.js

```diff
@@ -95,6 +94,28 @@ Meteor.methods({
 
 
 Tinytest.addAsync(
+    "livedata server - onMessage hook",
+    function (test, onComplete) {
+
+        var cb = Meteor.onMessage(function (msg, session) {
+            test.equal(msg.method, 'livedata_server_test_inner');
+            cb.stop();
+            onComplete();
+        });
+
+        makeTestConnection(
+            test,
+            function (clientConn, serverConn) {
+                clientConn.call('livedata_server_test_inner');
+                clientConn.disconnect();
```

Comment:
I believe you need to call `onComplete()` at the end of this callback. See the other `Tinytest.addAsync` tests below for examples.